controllers/kogitoserverlessworkflow_controller.go

```diff
@@ -115,7 +116,7 @@ func buildEnqueueRequestsFromMapFunc(c client.Client, build *operatorapi.KogitoS
 	for i := range list.Items {
 		workflow := &list.Items[i]
 
-		match, err := utils.SameOrMatch(build, workflow)
+		match, err := sameOrMatch(build, workflow)
 		if err != nil {
 			log.Errorf(err, "Error matching workflow %q with build %q", workflow.Name, build.Name)
```

Comment:
the error message returned by this func is pretty similar, maybe just return the error and print the error message in the sameOrMatch method?

Reply:
This is from an old PR. Sure, I'll remove the log from the client.